envsetup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# if file already exists,
+# confirm before overwrite
+if [ -f origami.env ]; then
+	read -p "origami.env already exists. Continue? (y/N): " -n 1 -r CONT
+	echo 
+	if [[ ! $CONT =~ ^[Yy]$ ]]; then
+		exit 0
+	fi
+fi
+
+echo "set -a" > origami.env
+
+# takes two arguments $1 and $2
+# first argument is env variable
+# second is its default value
+# function takes input and adds it
```

Comment:
What do we need this for?

Reply:
Since the origami.env file is in the .gitignore now, I thought it'd make sense for a way to generate the file for first time setup. Either that, or another solution suggested was providing an origami.env.default and telling users to copy it to origami.env and editing it in the setup. If you want I can edit this PR to do that.